core/scripts/Voting.js

```diff
@@ -410,10 +428,11 @@ class SendgridNotifier {
 }
 
 class VotingSystem {
-  constructor(voting, account, notifiers) {
+  constructor(voting, account, notifiers, maxBatchTransactions) {
```

Comment:
I think its better to separate between commit and reveal for now, the library is more so to move the constant itself to the root of `protocol` so that both `voter-dapp` and `AVS` can reference the same variable